ci-operator/jobs/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-crd-presubmits.yaml

```diff
@@ -0,0 +1,236 @@
+presubmits:
+  openshift/cluster-api-provider-aws:
+  - agent: jenkins
+    always_run: true
+    branches:
+    - ^crd$
+    context: ci/jenkins/e2e
+    labels:
+      master: ci.openshift.redhat.com
+    name: pull-ci-openshift-cluster-api-provider-aws-crd-e2e
+    optional: true
+    rerun_command: /test e2e
+    trigger: ((?m)^/test( all| e2e),?(\s+|$))
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - crd
+    context: ci/prow/e2e-aws
```

Comment:
This will run latest installer, with mao with this image which will fail, right?
So I think we don't want to run this job on this branch.
- Once the actuators CRD PRs are ready to merge:
- We create a PR on mao updating images-reference to point to CRD images
- Once that's green, we merge it and we merge actuators PRs
- Finally we update mao images-references to point to master actuators
make sense? cc @smarterclayton